pyproject.toml

```diff
@@ -1,3 +1,12 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = [
+    "setuptools<60.0",
```

Comment:
Is there a specific reason for this?
And why not use `setuptools==60.0`?
Also, [more recent versions are available](https://pypi.org/project/setuptools/#history).

Reply:
I simply stuck to scikit-learn for this one, but we could definitely try to use it for more recent versions.